### ISSUE TITLE
Slideshows and slider slideshows do not work on Safari

### ISSUE THREAD

**Issue by dtsiapas** [netrivet/prophoto-issues#378]
On this page I have a slideshow with crossfade
http://www.tsiapas.com/blog/φωτογράφος-γάμου-wedding-photographer-wedding-portfolio/
It works properly on IE and Chrome
It doesn't load on Safari
It doesn't playback properly with vertical images on Firefox

On the following page i have three sliders
http://www.tsiapas.com/blog/%CE%AC%CE%BB%CE%BC%CF%80%CE%BF%CF%85%CE%BC-%CE%B3%CE%AC%CE%BC%CE%BF%CF%85-wedding-albums/
It works properly on IE, Chrome and Firefox (haven't tested vertical images in slider on Firefox)
It doesn't load on Safari

**Comment by brianium** [comment]
@dtsiapas I can't seep to reproduce the safari issue. What version of Safari are you using? I do see the Firefox issue. We recently removed some code that was handling an issue with Firefox because it appeared to have been fixed in the latest Firefox. I will double check the Firefox issue

**Comment by dtsiapas** [comment]
I tried it on Safari 5.1.7 on windows 10, perhaps there is an updated version by now.
Also, i replicated the the problem on my android phone with Safari for android.

**Comment by dtsiapas** [comment]
@brianium I updated the link where i have the sliders on my first post, i had the wrong link! Please check this too

**Comment by brianium** [comment]
Ah. Safari 5.1.7 is about 4 versions behind now, so we would not be supporting that. I don't think Safari for Android is an official apple product is it? I think Safari on Android is a community project? I will definitely check out the updated link :)

**Comment by dtsiapas** [comment]
You are right but i didn't have an apple device to test them. Safari for windows is far left behind in development and the android version is a community project. I just hope that apple users can see the site properly.
Looking forward for the fix about firefox (currently i am running the latest firefox version 44.0.2)

**Comment by brianium** [comment]
@dtsiapas I looked at the other galleries too, and they seem to be working great. As far as our testing has gone, true apple devices and browsers look great :)

Apple stopped supporting Windows as of Safari 6.0, and the 5.1.7 version is lacking support for responsive images which would explain why they do not load.

Good catch on the Firefox bug though! That was a legitimate bug, and I just patched our galleries with a fix. That fix should be in with the next auto-update. Thank you for opening the issue!
